department-of-veterans-affairs/caseflow
# Hearing prep update: remove select for SLOGID

connects #3217 

Steps to Validate:
1. Go to the daily docket page
2. Update any field, observe the logs to ensure select is not there.
